ansible/provision-tls-cluster.yml

```diff
@@ -0,0 +1,29 @@
+# Creates a TLS enabled cluster with local CA in artifacts/ca for testing purposes
+# !!! WARNING !!!
+# If you intend to use this for a production cluster you will need to set handle_certs to false and provide your own certs
+# Using our demo CA for production is a bad idea :S
```

Comment:
change to
```
Our demo CA should not be used for production or your cluster will be insecure!
```

Reply:
Sorry, why is this insecure? We shouldn't use language like that.

Reply:
In fairness it's relatively true. A self signed cert from a CA on your local laptop is super bad practice and not what I think anyone would consider secure. Plus we aren't exactly doing much to set the local CA up for success

Reply:
There's nothing insecure about it. It creates properly formatted TLS certificates for use with fully encrypted connections.

The bad part of this is that you have to manage client CA integration manually. If you want to focus on that as the challenge for production environments, no problem. But it's not insecure.

Reply:
Does the generated CA have SANS properly setup?

If so, then I can see the argument for this being secure enough -- it's common enough to generate a CA & the certs and then put the CA into a vault.

Reply:
Let's just say "Using our demo certificate authority for production is not recommended, and is intended only for testing and proof of concepts. Redpanda encourages you to integrate with your corporate certificate management setup."

Reply:
done